06_PRC_Handson_User-Training.md

```diff
@@ -52,6 +52,14 @@ with iRODSSession(irods_env_file=env_file, **ssl_settings) as session:
 
 ### Using PRC on a Windows Machine
 
+>   
+> **Version warning:**: Version 1.1.2 and 1.1.3 of the Python-iRODSclient are not compatible with Windows.    
```

Comment:
the Python-iRODSclient should have space before 'client'. But I would write only PRC because full naming of this abbreviation exists in line 8.